emhttp/plugins/dynamix.vm.manager/novnc/app/styles/base.css

```
@@ -546,356 +555,352 @@ html {
-  max-width: 100%;
-  /* minus approximate height of title, height of subtitle, and margin */
-  max-height: calc(100vh - 10em - 25px);
+    box-sizing: border-box;
+    max-width: 100%;
+    /* minus approximate height of title, height of subtitle, and margin */
+    max-height: calc(100vh - 10em - 25px);
 }
 
 /* Settings */
 #noVNC_settings {
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)